src/openrpc/certificationextension.json

```diff
@@ -0,0 +1,383 @@
+                "schema": {
+                    "type": "boolean"
+                }
+            },
+            "examples": [
+                {
+                    "name": "Interact with the app and go to Menu screen",
+                    "params": [
+                        {
+                            "name": "intent",
+                            "value": {
+                                "action": "button",
+                                "data": {
+                                    "operation": "menu"
+                                },
+                                "context": {
+                                    "source": "voice"
+                                }
+                            }
+                        }
+                    ],
+                    "result": {
+                        "name": "success",
+                        "value": true
+                    }
+                }
+            ]
+        },
+        {
+            "name": "screenCapture",
+            "tags": [
+                {
+                    "name": "capabilities",
+                    "x-uses": [
+                        "xrn:firebolt:capability:certification:screencapture"
+                    ]
+                }
+            ],
+            "summary": "Fetch the current screen on the device and upload to the given server",
+            "params": [
+                {
+                    "name": "uploadPath",
+                    "required": false,
+                    "summary": "An optional `path` where screenshot can be uploaded. If not provided, base64 encoded image stream will be returned in the response.",
+                    "schema": {
+                        "type": "string"
+                    }
+                }
+            ],
+            "result": {
+                "name": "success",
+                "summary": "whether the call was successful or not",
+                "schema": {
+                    "oneOf": [
+                        {
+                            "type": "boolean"
+                        },
+                        {
+                            "type": "string"
+                        }
+                    ]
+                }
+            },
+            "examples": [
+                {
+                    "name": "Screen capture and sends back the image stream with base64 encoded",
+                    "params": [],
+                    "result": {
+                        "name": "iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mP8/wcAAgMBApDPF90AAAAASUVORK5CYII=",
+                        "value": true
+                    }
+                },
+                {
+                    "name": "Screen capture and sends back the status",
+                    "params": [
+                        {
+                            "name": "uploadPath",
+                            "value": "http://example.com/upload"
```

Comment:
How would the upload actually work?  Wouldn't this functionality require additional details, such as HTTP auth headers, etc?